packages/commonwealth/client/scripts/state/api/topics/fetchTopics.ts

```diff
@@ -12,31 +8,21 @@ interface FetchTopicsProps {
   includeContestData?: boolean;
 }
 
-const fetchTopics = async ({
-  communityId,
-  includeContestData = false,
-}: FetchTopicsProps): Promise<Topic[]> => {
-  const response = await axios.get(`${SERVER_URL}${ApiEndpoints.BULK_TOPICS}`, {
```

Comment:
we should update the `BULK_TOPICS` query cache invalidate references to now be invalidated by trpc utils ex:

https://github.com/hicommonwealth/commonwealth/blob/1bc7952382c77fb3269c4203d2aa49e9f6f72705/packages/commonwealth/client/scripts/state/api/topics/createTopic.ts#L7-L9

there are 3-4 of these in `state/api/topics/{here}`

Reply:
good catch!, fixed in last commit